images/fcos-bastion-image/root/usr/bin/prune_nodes

```diff
@@ -1,216 +1,11 @@
-  wait_for_node_status
-}
-
-function exec_on_nodes() {
-  action=${1}
-  shift
-  params=("${@}")
-  for bmhost in $(yq e -o=j -I=0 '.[]' "${SHARED_DIR}/hosts.yaml"); do
-    # shellcheck disable=SC1090
-    . <(echo "$bmhost" | yq e 'to_entries | .[] | (.key + "=\"" + .value + "\"")')
-    if [ ${#bmc_address} -eq 0 ] || [ ${#bmc_user} -eq 0 ] || [ ${#bmc_pass} -eq 0 ] || [ ${#name} -eq 0 ]; then
-      echo "Error while unmarshalling hosts entries"
-      exit 1
-    fi
-
-    # Execute the action
-    "${action}" "${bmc_address}" "${bmc_user}" "${bmc_pass}" "${name}" "${pdu_uri:-}" "${params[@]}" &
-
-  done
-  wait
-}
-
-############
- ## Main ##
-############
-
-if [ "$#" -ne 1 ]; then
-    echo "<3>Usage: $0 <cluster-name>"
-    exit 1
+if [ -z "${1}" ]; then
+  echo "Usage: $0 <cluster-name>"
+  exit 1
 fi
 
-CLUSTER=${1}
-INTERNAL_NET=${INTERNAL_NET:-192.168.80.0/24}
-DOCKER_NET=${DOCKER_NET:-172.17.0.0/16}
-SHARED_DIR="/var/builds/${CLUSTER}"
-conf_files=(/opt/dhcpd/root/etc/dnsmasq.conf
-            /opt/bind9_zones/zone
-            /opt/bind9_zones/internal_zone.rev
-            /opt/bind9_zones/external_zone.rev
-            /etc/vips_reserved
-            /etc/hosts_pool_reserved)
-
-if [ -s "${SHARED_DIR}"/hosts.yaml ]; then
-  # Remove firewall rules for bootstrap
-  ## Remove port forwarding from bastion host to bootstrap host
-  ## Remove bootstrap host access limit to internet if in disconnected network
-  ## Remove cluster nodes access limit to internet if in disconnected network
-  firewall_cleanup "${CLUSTER}"
-
-  # Delete VLAN on Juniper
-  vlan_cleanup "${CLUSTER}"
-
-  # Reboot any stuck node pdu
-  exec_on_nodes reboot_stuck_pdu 'for pdu reboot' "' on| off'"
-
-  # Wipe disk and power off
-  exec_on_nodes wipe_disk 'to power off' 'Power is off'
-
-else
-  echo "<4>${SHARED_DIR}/hosts.yaml file does not exist. Skipping firewall, nodes and vlan cleanup"
-fi
-
-# Delete provisioning network
-nmcli connection delete "${CLUSTER}"-provisioning-dev
-nmcli connection delete br-"${CLUSTER}"-provisioning
-
-# Destroy HAProxy network for $CLUSTER
-# Delete haproxy container if it exists for the cluster
-haproxy_cleanup "${CLUSTER}"
-
-# Clear cluster entries from the conf files for dhcp, dns, reserved vips if any
-# Deallocate reserved nodes for the cluster
-sed -i "/${CLUSTER}/d" "${conf_files[@]}"
-
-# Clean up directories used for cluster build
-## Clean up HAProxy, cluster, nfs, ignition configuration
-## Delete grubcfg files cluster directory in /opt/tftpboot
-## Delete cached openshift-images in /var/lib/libvirt/openshift-images/
-cluster_files_cleanup "${CLUSTER}"
-
-# Restart dnsmasq container
-docker restart dhcpd
-
-# Reload and flush bind9 container
-docker exec bind9 rndc reload && \
-  docker exec bind9 rndc flush
-
-echo "<4>$CLUSTER - Clean-up, wipe disks and deprovisioning completed successfully"
+podman run -it -v "/root/.ssh/id_rsa:/var/builds/$1/ssh-key:z" -v "/var/builds/:/var/builds/:Z" \
```

Comment:
Would it be better for the second volume be restricted to the respective container?
```-v "/var/builds/$1:/var/builds/$1:Z"```

Reply:
As that folder is going to be deleted before the container quits, I wanted to avoid it.